doc/source/index.rst

```diff
@@ -7,6 +7,8 @@
 Welcome to the OpenShift SDN Developer Docs
 ============================================
 
+Note: This is now outdated. Please see OpenShift SDN/OVN Kubernetes READMEs and CNOs README/wiki pages at their respective OpenShift GitHub repositories for the latest information.
+
```

Comment:
I mean, it' s just a nit, but adding an hyperlink to where the latest content lives now would clarify things.

Reply:
Totally right. I should have added it from the beginning :)